roseau/load_flow/typing.py

```diff
@@ -80,6 +80,7 @@
     | Q_[Sequence[complex | float]]
     | Sequence[complex | float | Q_[complex | float]]
 )
+ComplexArrayLikeScalarOr1D: TypeAlias = ComplexArrayLike1D | Q_[complex | float] | complex
```

Comment:
` | float` at the end of this type?

Reply:
`complex` is equivalent to `complex | float | int`